docs/content/en/docs/configuration/command-line.md

```diff
@@ -315,12 +380,19 @@ Configures an endpoint with statistics, debugging and health checks. The followi
 
 Options:
 
+* `--health-check-path`: Defines the URL to be used as a health check for the default server.  Defaults to `/healthz`.
 * `--healthz-port`: Defines the port number haproxy-ingress should listen to. Defaults to `10254`.
 * `--profiling`: Configures if the profiling URI should be enabled. Defaults to `true`.
 * `--stats-collect-processing-period`: Defines the interval between two consecutive readings of haproxy's `Idle_pct`, used to generate `haproxy_processing_seconds_total` metric. haproxy updates Idle_pct every `500ms`, which makes that the best configuration value, and it's also the default if not configured. Values higher than `500ms` will produce a less accurate collect. Change to 0 (zero) to disable this metric.
 
 ---
 
+## --sync-period
+
+Specifies how often to relist and confirm cloud resources. Defaults to 10 minutes.
```

Comment:
This description is also a bit mysterious to me 😅

Reply:
Yeah same here. This value is passed straight to the k8s client's informer factory and it's in fact a bit of mysterious to me as well what it's really doing. Need to dig a bit more in the code. Anyway I changed the description to something a bit more concrete even though it still doesn't explain the functionality.